org.uqbar.project.wollok.lib/src/wollok/lang.wlk

```diff
@@ -980,31 +990,135 @@ class Dictionary {
 }
 
 /**
+ *
+ * In Wollok we have numbers as immutable representation. You can customize
+ * how many decimals you want to work with, and printing strategies. So
+ * number two could be printed as "2", "2,00000", "2,000", etc.
+ *
+ * You can coerce a parameter or a result to integer, in that case
+ * you can customize the coercing strategy: rounding up, down or throwing an error
+ * if reference points to a non-integer value.
  *
  * @author jfernandes
+ * @author dodain (unification between Double and Integer in a single Number class)
+ *
  * @since 1.3
  * @noInstantiate
  */	
 class Number {
-	method +(other)
-	method -(other)
-	method *(other)
-	method /(other)
 
-	method >(other)
-	method >=(other)
-	method <(other)
-	method <=(other)
+	/** @private */
+	override method simplifiedToSmartString(){ return self.stringValue() }
+	
+	/** @private */
+	override method internalToSmartString(alreadyShown) { return self.stringValue() }
+	
+	/** @private */
+	method checkNotNull(value, operation) native
+	
+	/** 
+	 * @private
+	 *
+	 * Applies coercing strategy to integer. If it is an integer, nothing happens. Otherwise, 
+	 * if it is a decimal, one of this coercing algorithms can be used
+	 * 
+	 * - self is rounded up to an integer
+	 * - self is rounded down to an integer
+	 * - an error is thrown
+	 */
+	method coerceToInteger() native
+	
+	/**
+	 * The whole wollok identity implementation is based on self method
```

Comment:
No entiendo este comentario

Reply:
Era un comentario "histórico". Lo actualicé a "Two references are identical if they are the same number"